Library/Formula/github-markdown-toc.rb

```diff
@@ -0,0 +1,19 @@
+class GithubMarkdownToc < Formula
+  desc "Easy TOC creation for GitHub README.md (in go)"
+  homepage "https://github.com/ekalinin/github-markdown-toc.go"
+  url "https://github.com/ekalinin/github-markdown-toc.go/archive/0.4.1.tar.gz"
+  sha256 "fb50c97b7a19152307ba142e6a9ee6f7bdb6659e3e4caa0fe38ffb69db7cf567"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    system "make", "get-deps"
```

Comment:
I don't think this is why it's failing, but we don't allow `go get` during the install step; the dependencies (https://github.com/ekalinin/github-markdown-toc.go/blob/master/Makefile#L21) should be included as `go_resources`; see [`hugo.rb`](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/hugo.rb) for an example.